README.md

```diff
@@ -1,2 +1,33 @@
 # hw01
+
 Repository for homework 01
+
+## Claire Brainard Bio
+
+Here is a list of things I enjoy doing:
+
+  * Baking Bread
+  
+  * Hiking
+  
+  * Camping
+  
+  * Traveling, esp. roadtrips!
+  
+  * Making Kombucha
+  
+  
+  **After** I finish up my Maters degree here at UChicago, I am moving back to *Southern California*!
+  
+### Below is a photo of my cat, Marlon:
+![](6E521B46-F262-4C2E-96A2-6DEB84E6B3F6.jpg)
+
+
+
+
+I also love me a CrunchWrap Supreme from Taco Bell.
+Here is a link to google images of those:
+[LINK](https://www.google.com/search?q=taco+bell+crunchwrap+supreme&client=safari&rls=en&source=lnms&tbm=isch&sa=X&ved=0ahUKEwi15PXum7fhAhVr4YMKHefDAi0Q_AUIDigB&biw=1440&bih=837&dpr=2)
```

Comment:
You don't have to specify a "LINK" here, you can simply do `Here is a [link](https://www.google.com/search?q=taco+bell+crunchwrap+supreme&client=safari&rls=en&source=lnms&tbm=isch&sa=X&ved=0ahUKEwi15PXum7fhAhVr4YMKHefDAi0Q_AUIDigB&biw=1440&bih=837&dpr=2) to google images of those:`